src/super_gradients/recipes/dataset_params/imagenet_dataset_params.yaml

```diff
@@ -12,4 +12,53 @@ imagenet_pca_aug: 0.0 # imagenet pca augmentation (applied only to trainset)
 train_interpolation: default # interpolation mode
 rand_augment_config_string: # randaugment config string (see super_gradients/training/datasets/auto_augment.py)
 random_erase_prob: 0.0 # random erase probability (applied only to trainset)
-aug_repeat_count: 0 # amount of repetitions (each repetition of an example is augmented differently) for a trainset example.
\ No newline at end of file
+aug_repeat_count: 0 # amount of repetitions (each repetition of an example is augmented differently) for a trainset example.
+
+
+# TODO: REMOVE ABOVE, HERE FOR COMPATIBILITY UNTIL WE REMOVE DATASET_INTERFACE
+# TODO: UNCOMMENT BELOW WHEN ABOVE IS REMOVED
+# Base recipe for ImageNet Datasets amd Dataloaders.
+#img_mean: [0.485, 0.456, 0.406] # mean for normalization
+#img_std: [0.229, 0.224, 0.225]  # std  for normalization
+
+train_dataset_params:
+  root: /data/Imagenet/train
+  transform:
```

Comment:
Lets align imagenet dataset with the rest by just passing transforms (i.e a list).
We can then wrap it up with Compost in ImagenetDataset's __init__

Reply:
Changed in 007ebe69dcfae3d8ded7874f243b7cbaab5c4b25